packages/cache/__tests__/tar.test.ts

```diff
@@ -75,7 +82,7 @@ test('gzip extract tar', async () => {
 
   expect(mkdirMock).toHaveBeenCalledWith(workspace)
   const tarPath = IS_WINDOWS
-    ? `${process.env['windir']}\\System32\\tar.exe`
+    ? `${process.env['PROGRAMFILES']}\\Git\\usr\\bin\\tar.exe`
```

Comment:
Can we reduce the repetition of writing the complete path everywhere? Like use variable that can work with the extrapolation.

Reply:
Let me try that